packages/project-config/src/paths.ts

```diff
@@ -213,11 +217,37 @@ export const getPaths = (BASE_DIR: string = getBaseDir()): Paths => {
   return paths
 }
 
+/**
+ * Returns the route hook for the supplied page path.
+ * Note that the page name doesn't have to match
+ *
+ * @param pagePath
+ * @returns string
+ */
+export const getRouteHookForPage = (pagePath: string | undefined | null) => {
+  if (!pagePath) {
+    return null
+  }
+
+  // We just use fg, so if they make typos in the routeHook file name,
+  // it's all good, we'll still find it
+  return fg
+    .sync('*.routeHooks.{js,ts,tsx,jsx}', {
+      absolute: true,
+      cwd: path.dirname(pagePath), // the page's folder
+    })
+    .at(0)
+}
+
+export const getAppRouteHook = () => {
+  return resolveFile(path.join(getPaths().web.src, 'App.routeHooks'))
+}
```

Comment:
I couldn't find these new functions being used anywhere (just did a simple search); what's the plan for them?

Reply:
I can remove these. They're used in the experimental branch. I just checked out stuff from there.